frontend2/src/components/BaseCard.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+interface BaseCardProps {
+  gameImage: string;
+  gameName: string;
+  gameDescr: string;
+  gameYear: number;
+}
+
+const BaseCard: React.FC<BaseCardProps> = ({ gameImage, gameName, gameDescr, gameYear }): JSX.Element => {
+  const redirect = (): void => {
+    console.log(gameYear.toString()); // Redirect to year page
+  };
+
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-md transition duration-300 ease-in-out transform hover:shadow-lg hover:cursor-pointer" onClick={redirect}>
+      <div className="aspect-w-4 aspect-h-3">
+        <img className="object-cover" src={require(`../${gameImage}`)} alt={gameName} />
```

Comment:
```suggestion
        <img className="object-cover" src={gameImage} alt={gameName} />
```
you don't need the require() here